app/components/GettingStartedModal/GettingStartedModal.tsx

```diff
@@ -27,6 +30,7 @@ const castedGettingStartedSchema = gettingStartedSchema as GettingStartedSchema;
 const GettingStartedModal = () => {
     const [modalOpen, setModalOpen] = useState(() => isGettingStartedModalDisabledInLocalStorage());
```

Comment:
Optional, fun code golf:

```suggestion
    const [modalOpen, setModalOpen] = useState(isGettingStartedModalDisabledInLocalStorage);
```

Reply:
yeah, good point!

`localStorage['key']` returns `undefined` and `localStorage.getItem('key')` returns null for lack of key

fixed